src/providers/credential/OpenId4VcIssuanceProvider.ts

```diff
@@ -324,16 +324,22 @@ class OpenId4VcIssuanceProvider {
     if (!this._issuerBranding) {
       this._issuerBranding = this._serverMetadata.credentialIssuerMetadata?.display;
     }
-
     if (!this._credentialBranding) {
       this._credentialBranding = new Map<string, Array<IBasicCredentialLocaleBranding>>();
       await Promise.all(
         this._credentialsSupported.map(async (metadata: CredentialSupported): Promise<void> => {
-          const localeBranding: Array<IBasicCredentialLocaleBranding> = await Promise.all(
-            (metadata.display ?? []).map(
-              async (display: CredentialsSupportedDisplay): Promise<IBasicCredentialLocaleBranding> =>
-                await ibCredentialLocaleBrandingFrom({localeBranding: await credentialLocaleBrandingFrom(display)}),
-            ),
+          const optionalLocaleBranding: Array<IBasicCredentialLocaleBranding | undefined> = await Promise.all(
+            (metadata.display ?? []).map(async (display: CredentialsSupportedDisplay): Promise<IBasicCredentialLocaleBranding | undefined> => {
+              try {
+                return await ibCredentialLocaleBrandingFrom({localeBranding: await credentialLocaleBrandingFrom(display)});
+              } catch (e) {
+                console.error(`Could not fetch branding. Branding data: ${JSON.stringify(display)}`);
```

Comment:
i am not sure if this is the way we should handle this. Also what situation did you have where the branding was not present?

Reply:
Images could be unavailable because of a network issue for instance. I rather have us accept the credential without branding then having an error because of branding issues.

If I remember correctly it was a case where the image could not be found, right @sander

Reply:
should we then not fix that in the plugin instead of the wallet?